build/release/containers.sh

```diff
@@ -75,22 +75,21 @@ ENTRYPOINT ["/usr/bin/rookd"]
 EOF
     build_artifact $os $arch $tmpdir rook/rookd rookd
     rm -fr $tmpdir
-
-    # TODO: build the client and toolbox for arm
-    [[ ${arch} == "amd64" ]] || return 0
  
     echo "Building the rook-client container"
     mkdir $tmpdir
     cat <<EOF > $tmpdir/Dockerfile
-FROM ubuntu
-RUN apt-get update && \
-    apt-get install -yq --no-install-recommends wget s3cmd kmod module-init-tools sudo
```

Comment:
It's not the existence of the `rbd` kernel module I'm curious about, it's how the `rook` client binary would be able to shell out to `modprobe` to load it.  Perhaps that's just built-in in the Alpine image too?  Did you happen to try mapping a block image using the `rook-client` pod where no one else had already loaded the `rbd` kernel module?